src/Modal/index.jsx

```diff
@@ -16,13 +20,21 @@ class Modal extends React.Component {
     this.setCloseButton = this.setCloseButton.bind(this);
 
     this.headerId = newId();
+    this.el = document.createElement('div');
+
+    if (!modalRoot) {
+      modalRoot = document.createElement('div');
+      modalRoot.id = modalRootId;
+      document.body.appendChild(modalRoot);
```

Comment:
This doesn't feel quite right to me but I can't think of a low-API way to ensure the element exists in the DOM, outside the bounds of this component.